doc/ug/dv_methodology.md

```diff
@@ -415,6 +415,28 @@ It is recommended to follow these guidelines when collecting coverage:
 *  Collect all coverage metrics (except toggle based on above bullet) on the DUT and all of its non-pre-verified sub-modules
 *  Treat pre-verified sub-modules as ['black-box'](#integration-testing) in terms of coverage collection
 
+### X-Propagation (Xprop)
+
+It's mandatory to enable Xprop when running regression for coverage closure.
+Xprop is enabled by default and we choose `tmerge` mode which is closer to actual hardware behavior and is the more commonly used mode.
+It's recommended to enable Xprop all the time since it only adds less than 10% simulation time.
+
+In order to test X propagation more effectively, we should implement driver to set input data/address/control signals to X when it's not in valid period.
+For example, when a_valid is 0 in TLUL interface, we drive data, address and control signals to unknown values.
+```systemverilog
+  function void invalidate_a_channel();
+    vif.host_cb.h2d.a_opcode  <= tlul_pkg::tl_a_op_e'('x);
+    vif.host_cb.h2d.a_param   <= '{default:'x};
+    vif.host_cb.h2d.a_size    <= '{default:'x};
+    vif.host_cb.h2d.a_source  <= '{default:'x};
+    vif.host_cb.h2d.a_address <= '{default:'x};
+    vif.host_cb.h2d.a_mask    <= '{default:'x};
+    vif.host_cb.h2d.a_data    <= '{default:'x};
+    vif.host_cb.h2d.a_user    <= '{default:'x};
+    vif.host_cb.h2d.a_valid   <= 1'b0;
+  endfunction : invalidate_a_channel
+```
+
```

Comment:
I think we should enforce this. Since we don't have DV guide in repo, I put it here. Will discuss with @sriyerg later

Reply:
Will prioritize on adding the DV style guide in soon.